docs/essentials/calendar.md

```diff
@@ -0,0 +1,167 @@
+---
+title: "Xamarin.Essentials: Calendar"
+description: "This document describes the Calendar class in Xamarin.Essentials, which lets you view the user's calendar and event information."
+ms.assetid: a6d5f2a2-0f49-4726-9a74-8e7667ddb127
+author: ScottBTR
+ms.author: ScottYoung-5180
+ms.date: 03/12/2019
+---
+
+# Xamarin.Essentials: Calendar
+
+The **Calendar** class lets you view the user's calendar and event information.
+
+## Get started
+
+[!include[](~/essentials/includes/get-started.md)]
+
+To access the **Calendar** functionality the following platform specific setup is required.
+
+# [Android](#tab/android)
+
+The `Read Calendar` and `Write Calendar` permission is required and must be configured in the Android project. 
+
+Open the **AndroidManifest.xml** file under the **Properties** folder and add the following inside of the **manifest** node.
+
+```xml
+<uses-permission android:name="android.permission.READ_CALENDAR" />
+<uses-permission android:name="android.permission.WRITE_CALENDAR" />
+```
+
+Or right click on the Android project and open the project's properties. Under **Android Manifest** find the **Required permissions:** area and check the **READ_CALENDAR** and **WRITE_CALENDAR** permission. This will automatically update the **AndroidManifest.xml** file.
+
+# [iOS](#tab/ios)
+
+Your app's **Info.plist** must contain the `NSCalendarsUsageDescription` and the `NSRemindersUsageDescription` key in order to access the device’s calendars & reminders.
```

Comment:
Can you confirm that for the limited api we're exposing that we need NSRemindersUsageDescription

Reply:
We do not, have removed accordingly